server/game/GameActions/GameAction.ts

```diff
@@ -25,30 +24,37 @@ export class GameAction {
     cost = '';
     effect = '';
     defaultProperties: GameActionProperties = { cannotBeCancelled: false, optional: false };
-    getDefaultTargets: (context: AbilityContext) => any = context => this.defaultTargets(context);
+    getDefaultTargets: (context: AbilityContext) => any = (context) => this.defaultTargets(context);
 
     constructor(propertyFactory: GameActionProperties | ((context?: AbilityContext) => GameActionProperties) = {}) {
-        if(typeof propertyFactory === 'function') {
+        if (typeof propertyFactory === 'function') {
             this.propertyFactory = <(context?: AbilityContext) => GameActionProperties>propertyFactory;
         } else {
             this.properties = propertyFactory;
         }
     }
 
-    defaultTargets(context: AbilityContext): any[] { // eslint-disable-line no-unused-vars
+    defaultTargets(context: AbilityContext): any[] {
+        // eslint-disable-line no-unused-vars
```

Comment:
This looks like it will break the `eslint-disable-line` directive. Is that intentional? This happens on a number of other lines as well.

Reply:
It's ok, cause the TS codebase is not validated by eslint